src/content/en/2020/accessibility.md

```diff
@@ -226,227 +346,247 @@ Some assistive technologies for the Web include:
 
 Screen readers present content audibly, usually by the computer speaking or announcing the content in the interface as the user navigates and interacts. This enables blind, low vision, and other disabled and non-disabled  users to consume the content without needing to rely on the visual cues displayed on the screen.
 
-### ARIA
+### Introduction to ARIA
 
-In 2014 the WAI published Accessible Rich Internet Applications, or ARIA. They [describe ARIA as](https://www.w3.org/WAI/standards-guidelines/aria/), “WAI-ARIA, the Accessible Rich Internet Applications Suite, defines a way to make Web content and Web applications more accessible to people with disabilities. It especially helps with dynamic content and advanced user interface controls developed with Ajax, HTML, JavaScript, and related technologies.”
+{# TODO (Authors) Aria was the only heading under "Assistive technologies on the web" (last year ARIA and Forms were two segments under that heading). So I changed "ARIA" to "Introduction to ARIA" and moved all the headings under it up a level to avoid this. Alternatiovely we could merge "Assistive technologies on the web" and "Introduction to Aria" into one section?  #}
```

Comment:
@oluoluoxenfree / @alextait1 let me know if you are OK with this

Reply:
Yeah I'm fine with it